### ISSUE TITLE
Cannot specify database with space in name in config

### ISSUE THREAD

**Issue by Bilge** [pgbouncer/pgbouncer#971]
Although the documentation seems to claim that simply quoting the dbname is sufficient, the following config will cause PgBouncer to cease to start up:

```ini
[databases]
"Foo Bar" = host=foo
```

> ERROR syntax error in configuration (/etc/pgbouncer/pgbouncer.ini:14), stopping loading

Nevertheless, in general, PgBouncer handles such database names just fine. We know this because the following config works:

```ini
[databases]
* = host=foo
```

Whereafter we can connect to a database so named `"Foo Bar"` or whatever else. It seems this is simply a bug in the configuration parser.

**Comment by JelteF** [comment]
Yes, this is a problem. This should be fixed in a libusual its cfparser: https://github.com/libusual/libusual/blob/master/usual/cfparser.c

Feel free to provide a PR on that repo.

**Comment by Bilge** [comment]
I think that may be beyond my ability.
